services/images/service.go

```diff
@@ -51,20 +52,20 @@ func (s *Service) Register(server *grpc.Server) error {
 func (s *Service) Get(ctx context.Context, req *imagesapi.GetImageRequest) (*imagesapi.GetImageResponse, error) {
 	var resp imagesapi.GetImageResponse
 
-	return &resp, s.withStoreView(ctx, func(ctx context.Context, store images.Store) error {
+	return &resp, errdefs.ToGRPC(s.withStoreView(ctx, func(ctx context.Context, store images.Store) error {
 		image, err := store.Get(ctx, req.Name)
 		if err != nil {
-			return mapGRPCError(err, req.Name)
+			return err
 		}
 		imagepb := imageToProto(&image)
 		resp.Image = &imagepb
 		return nil
-	})
+	}))
 }
 
 func (s *Service) Update(ctx context.Context, req *imagesapi.UpdateImageRequest) (*imagesapi.UpdateImageResponse, error) {
 	if err := s.withStoreUpdate(ctx, func(ctx context.Context, store images.Store) error {
-		return mapGRPCError(store.Update(ctx, req.Image.Name, descFromProto(&req.Image.Target)), req.Image.Name)
+		return errdefs.ToGRPC(store.Update(ctx, req.Image.Name, descFromProto(&req.Image.Target)))
```

Comment:
inconsistent with other changes, move `ToGRPC` on the return of the function not the closure. Same down below in this file